pkg/operator/ceph/csi/peermap/config.go

```diff
@@ -351,12 +351,32 @@ func decodePeerToken(token string) (*cephclient.PeerToken, error) {
 }
 
 func getPeerPoolDetails(ctx *clusterd.Context, args ...string) (cephclient.CephStoragePoolDetails, error) {
-	peerPoolDetails, err := ctx.Executor.ExecuteCommandWithTimeout(exec.CephCommandsTimeout, "ceph", args...)
+
+	poolDetailsTempFile, err := os.CreateTemp("/tmp/output.json", "")
+	if err != nil {
+		return cephclient.CephStoragePoolDetails{}, errors.Wrapf(err, "failed to generate temporarily file %q", poolDetailsTempFile.Name())
+	}
```

Comment:
Use `os.CreateTemp` with the different patterns as if we have multiple pools/reconciles. Each reconcile should use its unique file

Reply:
should that matter if the file is removed (using `defer`) after using it?

Reply:
This looks fine